PHPFIleHandlingAndSessions/deleteContent.php

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Edit Content</title>
+</head>
+<body>
+	<form name="modify" method="post" action="delete.php">
+		<?php
+			session_start();
+			if(!isset($_SESSION['username']))
+			{
+				@header("location:loginScreen.html");
+			}
+			$username=$_SESSION['username'];
+			try
+			{
+				$db=new PDO('mysql:host=localhost;port=3306;dbname=BooksDetails','root','root');
+				$res=$db->query("select * from books where username='".$username."'",PDO::FETCH_ASSOC);
+
+				foreach ($res as $result) 
```

Comment:
Check if the data is not empty before going into the loop. Check and apply it everywhere